packages/react/jest.config.js

```diff
@@ -1 +1,4 @@
-module.exports = require('../../jest.config');
+module.exports = {
+  ...require('../../jest.config'),
+  setupFilesAfterEnv: ['<rootDir>/../../test/jest.setup.ts'],
```

Comment:
Oh, was the setup for tests not working? 🙈

Reply:
When running the tests in Intellij, `<rootDir>` is set to the `react` directory, so I had to override `setupFilesAfterEnv` here to make it work.

Reply:
Ah, thats good to know! Still works fine both places for vscode so go for it 👍